src/prob/tp_mld.jl

```diff
@@ -98,52 +98,52 @@ function post_tp_mld(pm::_PMs.GenericPowerModel)
     end
 
     for i in _PMs.ids(pm, :trans)
-        constraint_tp_trans(pm, i)
+        constraint_mc_trans(pm, i)
     end
 
     objective_tp_min_load_delta(pm)
 end
 
 
 "Load shedding problem with storage"
-function post_tp_mld_strg(pm::_PMs.GenericPowerModel)
-    variable_tp_indicator_bus_voltage(pm; relax=true)
-    variable_tp_bus_voltage_on_off(pm)
+function post_mc_mld_strg(pm::_PMs.AbstractPowerModel)
+    variable_mc_indicator_bus_voltage(pm; relax=true)
+    variable_mc_bus_voltage_on_off(pm)
 
-    variable_tp_trans_flow(pm)
-    variable_tp_branch_flow(pm)
+    variable_mc_trans_flow(pm)
+    variable_mc_branch_flow(pm)
 
-    variable_tp_storage(pm)
+    variable_mc_storage(pm)
 
-    variable_tp_indicator_generation(pm; relax=true)
-    variable_tp_indicator_storage(pm; relax=true)
+    variable_mc_indicator_generation(pm; relax=true)
+    variable_mc_indicator_storage(pm; relax=true)
     for c in _PMs.conductor_ids(pm)
         _PMs.variable_generation_on_off(pm, cnd=c)
-        variable_tp_on_off_storage(pm, cnd=c)
+        variable_mc_on_off_storage(pm, cnd=c)
         _PMs.variable_dcline_flow(pm, cnd=c)
     end
 
-    variable_tp_indicator_demand(pm; relax=true)
-    variable_tp_indicator_shunt(pm; relax=true)
+    variable_mc_indicator_demand(pm; relax=true)
+    variable_mc_indicator_shunt(pm; relax=true)
 
-    constraint_tp_model_voltage(pm)
+    constraint_mc_model_voltage(pm)
 
     for i in _PMs.ids(pm, :ref_buses)
-        constraint_tp_theta_ref(pm, i)
+        constraint_mc_theta_ref(pm, i)
     end
-    constraint_tp_bus_voltage_on_off(pm)
+    constraint_mc_bus_voltage_on_off(pm)
 
     for i in _PMs.ids(pm, :gen)
         _PMs.constraint_generation_on_off(pm, i)
     end
 
     for i in _PMs.ids(pm, :bus), c in _PMs.conductor_ids(pm)
-        constraint_tp_power_balance_shunt_trans_shed(pm, i, cnd=c)
+        constraint_mc_power_balance_shunt_trans_shed(pm, i, cnd=c)
     end
 
     for i in _PMs.ids(pm, :storage)
         _PMs.constraint_storage_state(pm, i)
-        constraint_tp_storage_exchange(pm, i)
+        constraint_mc_storage_exchange(pm, i)
```

Comment:
I think the loss and complementarity constraints from PowerModels can be used directly here instead of calling this function.